db/seeds.rb

```diff
@@ -17,7 +17,55 @@
   insurance: 'Yes',
   can_accomodate_meetings: true,
   areas_of_interest: ['Providing ongoing assistance to sponsors through SSP matching'],
-  comments: ''
+  comments: '',
+  private_notes: 'Private notes'
 })
-
 puts "Created lawyer #{lawyer.email} with password 'test'"
+
+student = Student.create!(name: "Example Student", email: "example_student@example.com", password: "test", approved: true, extra: {
+  approved: true,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.